src/components/Form.js

```diff
@@ -6,17 +6,29 @@ import {
   Field as FormikField,
   ErrorMessage as FormikErrorMessage
 } from 'formik';
+import media from '../utils/media';
+
 
 
 export const Form = styled(FormikForm)`
   display: flex;
   flex-direction: column;
   align-items: center;
+  min-height: calc(100vh - 32px);
+    ${media.desktop`
```

Comment:
@aayushineeshu  The desktop media query should come after tablet ..else the tablet query takes precedence. I suspect that in this case, even for desktop the min-height is coming from tablet query. can you confirm?